presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.235
+=============
+
+General Changes
+_______________
+* Fix named queries where its output does not match its column list (:issue:`14333`).
+* Fix error in analysis phase.
+* Fix ZSTD compression issue with zero row file for missing buckets.
+* Improve performance of ``UNNEST``.
+* Improve ``CREATE FUNCTION`` to allow parameter type list and return type to have a length up to 30k characters.
+* Add the ability to pre-check method bytecode size is within limits.
```

Comment:
What about
```
Add support for pre-checking if bytecode size is within limits.
```

Reply:
I think that's still too low level for users.  Maybe:
```
Fix issue where queries with large filters in the where clause would cause full GCs.  Now such queries will fail quickly with <expected exception>
```

Reply:
Technically, it _may_ not cause full gc.  How about

```
* Fix issue where queries with large filters in the where clause would cause excessive memory usage.  Now such queries will fail quickly with ``GENERATED_BYTECODE_TOO_LARGE``.
```